release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -3155,3 +3155,58 @@ $ oc adm release info 4.13.11 --pullspecs
 ==== Updating
 
 To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-13-12"]
+=== RHBA-2023:5011 - {product-title} 4.13.12 bug fix
+
+Issued: 2023-09-12
+
+{product-title} release 4.13.12 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:5011[RHBA-2023:5011] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:5014[RHBA-2023:5014] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.13.12 --pullspecs
+----
+
+[id="ocp-4-13-12-features"]
+==== Features
+
+[id="ocp-4-13-12-networking-sriov-exclude-topology"]
+===== Exclude SR-IOV network topology for NUMA-aware scheduling
+
+With this release, you can exclude advertising the Non-Uniform Memory Access (NUMA) node for the SR-IOV network to the Topology Manager. By not advertising the NUMA node for the SR-IOV network, you can permit more flexible SR-IOV network deployments during NUMA-aware pod scheduling.
+
+For example, in some scenarios, it is a priority to maximize CPU and memory resources for a pod on a single NUMA node. By not providing a hint to the Topology Manager about the NUMA node for the pod’s SR-IOV network resource, the Topology Manager can deploy the SR-IOV network resource and the pod CPU and memory resources to different NUMA nodes. In earlier {product-title} releases, the Topology Manager attempted to place all resources on the same NUMA node only.
+
+For more information about this more flexible SR-IOV network deployment during NUMA-aware pod scheduling, see xref:../networking/hardware_networks/configuring-sriov-device.adoc#nw-sriov-exclude-topology-manager_configuring-sriov-device[Exclude the SR-IOV network topology for NUMA-aware scheduling].
```

Comment:
FYI, the page that this xref points to doesn't mention NUMA-aware pod scheduling. Is it the right page?